### PR TITLE
Adding accordion documentation

### PR DESCRIPTION
**Pull request by colinpmacarthur**

### CODE REVIEW

File: _components/accordions.md [hunk 1]
```diff
@@ -139,29 +139,21 @@ lead: Intro text on what is included in this section and how to use it. No more
     Documentation
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
-    <h4 class="usa-heading">Use</h4>
+    <h4 class="usa-heading">Usability</h3>
```
Comment (msecret) [inline _components/accordions.md]: Should this be an h4 or an h3? Right now you have `<h4>` as the opening tag and `</h3>` as the closing tag.

File: _components/accordions.md [hunk 2]
```diff
@@ -139,29 +139,21 @@ lead: Intro text on what is included in this section and how to use it. No more
...
+    <h5>When to use</h4>
```
Comment (msecret) [inline _components/accordions.md]: Same as above but this time h5 and h4.

So this should either be `<h5>When to use</h5>` or `<h4>When to use</h4>` depending on what's needed for the design.

File: _components/accordions.md (hunk 1 above)
Comment (colinpmacarthur) [inline _components/accordions.md]: Good catch. Fixed in 47985e2.

File: _components/accordions.md (hunk 2 above)
Comment (colinpmacarthur) [inline _components/accordions.md]: Good catch. Fixed in 47985e2.

**Pull request merged**